src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java

```diff
@@ -32,30 +32,30 @@
     /**
      * Delete through link option.
      */
-    public void clickOnDeleteProjectLink() {
+    public void clickOnDeleteWorkspaceProjectLink() {
```

Comment:
please restore before name

Reply:
This method was moved to respective Workspace/Project settings page